packages/components/src/core/InputSearch/index.tsx

```diff
@@ -100,8 +128,9 @@ const InputSearch = forwardRef<HTMLDivElement, InputSearchProps>(
           sdsStyle={sdsStyle}
           sdsStage={hasValue ? "userInput" : "default"}
           onChange={handleChange}
-          onKeyPress={handleKeyPress}
+          onKeyDown={handleKeyPress}
           intent={intent}
+          autoComplete="off"
```

Comment:
This prevents the browser's default auto completion menu from being displayed for the InputSearch.

<img width="288" alt="Screenshot 2023-09-21 at 5 08 16 PM" src="https://github.com/chanzuckerberg/sci-components/assets/927990/c0c91b69-ab72-4359-98f5-3de55d9add3d">

Reply:
Let's add this comment in the code too for context!